packages/hoprd/src/index.ts

```diff
@@ -67,6 +67,11 @@ const argv = yargs
     describe: 'A password to encrypt your keys',
     default: ''
   })
+  .option('apiToken', {
+    describe: 'A REST API token and admin panel password for user authentication',
+    string: true,
+    default: ''
```

Comment:
Does a default value of `''` mean that we need to pass a zero-length string on every request if `--apiToken` is *not* set?